### DISCUSSION TITLE
Config in yaml

### DISCUSSION THREAD

**Discussion by kklecho** [specmatic/specmatic#1079] [Ideas and Suggestions]
Allow specmatic configs in yaml format.

I'm new adopter. Switching between yaml (schema) and json for specmatic can feels confusing.

**Comment by joelrosario** [comment]
Thanks for reaching out @kklecho. That's a good idea. We'll add it to our backlog.

**Comment by yogeshnikam671** [comment]
Hey @kklecho, the support for config in yaml has been added in Specmatic v2.0.x onwards. Please try it out and let us know your feedback if any.
[Here](https://specmatic.io/documentation/configuration.html) is the documentation for the same.
